resources/css/main.css

```diff
@@ -0,0 +1,684 @@
+        top:20px;
+        -webkit-transition: all 0.1s ease-in-out;
+        -moz-transition: all 0.1s ease-in-out;
+        -o-transition: all 0.1s ease-in-out;
+        -ms-transition: all 0.1s ease-in-out;
+        transition: all 0.1s ease-in-out;
+        
+        
+    }
+    .image-container{
+        margin:auto;
+        width:80%;
+        max-width:1300px;
+        
+    }
+    .tour-dropdown-toggle {
+        width:80%;
+        text-align:center;
+    }
+    .tour-dropdown-menu{
+        width:80%;
+        margin-left:10%;
+        
+    }
+    .navbar-brand {
+      font-size: 30px;
+    }
+    .desktop-hide{
+        display:none;
+        
+    }
+    .csh-body{
+        padding-top:20px;
+        padding-left:50px;
+        padding-right:50px;
+        
+    }
+    .sponsor-image-container{
+        width:25%;
+        display: inline-block;
+        
+        
+    }
+    .traditions-page-thumbnail{
+        margin-top:30px;
+        
+    
+    }
+    
+    .sponsor-image{
+        width:90%;
+        max-width:200px;
+        margin-bottom:40px;
+       border-radius:6px; position:relative;
+        left: 50%;
+	   -webkit-transform: translate(-50%, 0%);
+	   -moz-transform: translate(-50%, 0%);
+	   -ms-transform: translate(-50%, 0%);
+	   -o-transform: translate(-50%, 0%);
+	   transform: translate(-50%, 0%);
+    }
+    .insights-btn{
+        font-size:25px;
+    }
+    
+    nav a{
+        transition: padding .1s ease-in, height .1s ease-in, font-size .1s ease-in, margin .1s ease-in;
+        -webkit-transition: padding .1s ease-in, height .1s ease-in, font-size .1s ease-in, margin .1s ease-in;
+        -moz-transition: padding .1s ease-in, height .1s ease-in, font-size .1s ease-in, margin .1s ease-in;
+        
+        
+    }
+    nav.navbar{
+        -webkit-transition: height 0.1s ease-in-out;
+        -moz-transition: height 0.1s ease-in-out;
+        -o-transition: height 0.1s ease-in-out;
+        -ms-transition: height 0.1s ease-in-out;
+        transition: height 0.1s ease-in-out;
+        
+    }
+
+    nav.navbar.shrink {
+      min-height: 35px;
+      background:rgba(255,255,255,.98);
+    }
+
+    nav.shrink a {
+      padding-top: 15px !important;
+      padding-bottom: 15px !important;
+      font-size: 15px;
+    }
+
+    nav.shrink .navbar-brand {
+      font-size: 25px;
+    }
+
+    nav.shrink .navbar-toggle {
+      padding: 4px 5px;
+      margin: 8px 15px 8px 0;
+    }
+    nav.shrink .dropdown-menu a{
+        padding-top:5px !important;
+        padding-bottom:5px !important;
+        font-size:14px;
+        
+    }
+    nav.shrink .csh-navbar-nav-shift{
+        position:relative;
+        top:0px;
+        
+        
+    }
+    .csh-jumbotron{
+        height:400px;
+        
+    }
+    .csh-page-header {
+        height:100px;
+    } 
+}
+.navbar-csh {
+    border-top:5px solid #E11C52;
+    background:#FFF;
+}
+
+.csh-logo {
+    width:150px;
+    margin-top:-20px;
```

Comment:
What if this was `-18px`? It might not be mathematically centered, but it appears aligned better to my eye. At minimum, `-19px` makes it base-aligned with the navigation menu items to the right of it. Screenshots below, best if viewed full size.

`-18px` Screenshot:
![screenshot 2015-12-12 01 25 18](https://cloud.githubusercontent.com/assets/1441807/11760610/49132b04-a06f-11e5-83b6-98937a75bd0c.png)


`-20px` (current) Screenshot:
![screenshot 2015-12-12 01 25 21](https://cloud.githubusercontent.com/assets/1441807/11760611/4d49e3ac-a06f-11e5-85ab-8e1c28b9f085.png)